w3-hw1-ThuyDung/cypress/pages/formPage.js

```diff
@@ -0,0 +1,245 @@
+            cy.get(this.TXT_FIRSTNAME).should('have.css', 'background-image', this.SYMBOL_VALID);
+        }else{
+            cy.get(this.TXT_FIRSTNAME).should('have.css', 'border-color', this.COLOR_RED);
+            cy.get(this.TXT_FIRSTNAME).should('have.css', 'background-image', this.SYMBOL_INVALID);
+        }
+        return this;
+    },
+    isLastNameValid(isValid){
+        if(isValid === true){
+            cy.get(this.TXT_LASTNAME).should('have.css', 'border-color', this.COLOR_GREEN);
+            cy.get(this.TXT_LASTNAME).should('have.css', 'background-image', this.SYMBOL_VALID);
+        }else{
+            cy.get(this.TXT_LASTNAME).should('have.css', 'border-color', this.COLOR_RED);
+            cy.get(this.TXT_LASTNAME).should('have.css', 'background-image', this.SYMBOL_INVALID);
+        }
+        return this;
+    },
+    isMobileValid(isValid){
+        if(isValid === true){
+            cy.get(this.TXT_MOBILE).should('have.css', 'border-color', this.COLOR_GREEN);
+            cy.get(this.TXT_MOBILE).should('have.css', 'background-image', this.SYMBOL_VALID);
+        }else{
+            cy.get(this.TXT_MOBILE).should('have.css', 'border-color', this.COLOR_RED);
+            cy.get(this.TXT_MOBILE).should('have.css', 'background-image', this.SYMBOL_INVALID);
+        }
+        return this;
+    },
+    isEmailValid(isValid){
+        if(isValid === true){
+            cy.get(this.TXT_EMAIL).should('have.css', 'border-color', this.COLOR_GREEN);
+            cy.get(this.TXT_EMAIL).should('have.css', 'background-image', this.SYMBOL_VALID);
+        }else{
+            cy.get(this.TXT_EMAIL).should('have.css', 'border-color', this.COLOR_RED);
+            cy.get(this.TXT_EMAIL).should('have.css', 'background-image', this.SYMBOL_INVALID);
+        }
+        return this;
+    },
+    isGenderValid(isValid){
+        for(let i=0; i< 3; i++){
+            cy.get(this.LBL_OPTION).eq(i).within(($el) => {
+                cy.window().then((win) => {
+                    const beforeElement = win.getComputedStyle($el[0], "::before");
+                    const afterElement = win.getComputedStyle($el[0], "::after");
+                    const borderColor = beforeElement.getPropertyValue("border-color");
+                    const color = afterElement.getPropertyValue("color");
+                    if(isValid === true){
+                      expect(borderColor).to.equal(this.COLOR_GREEN);
+                      expect(color).to.equal(this.COLOR_GREEN);
+                    }else{
+                        expect(borderColor).to.equal(this.COLOR_RED);
+                        expect(color).to.equal(this.COLOR_RED);
+                    }
+                })
+            })
+        }
+        return this;
+    },
+
+
+
+
+
+//After regiter successfully => show popup
+    popup :{
+        TXT_NAME_POPUP: "table > tbody > tr:nth-child(1) > td:nth-child(2)",
```

Comment:
Have you tried` TXT_VALUE: "td + td"` then `cy.get(this.TXT_VALUE).contains(value).should("be.visible")`? You can use just one locator to implement all verify because you know how the result should be